packages/create-plugin/templates/scenes-app/provisioning/custom.ini

```diff
@@ -0,0 +1,6 @@
+app_mode = development
+
+[feature_toggles]
+topnav=true
```

Comment:
I think we can get rid of this, `topnav` is [already enabled by default](https://github.com/grafana/grafana/blob/323ee7c38ceb18b8e71c780d797fabac7041a673/pkg/services/featuremgmt/registry.go#L179).